src/components/assign-lesson-drv/scss/_lessonList.scss

```diff
@@ -141,8 +141,9 @@ lesson-list {
                     /* Base for label styling */
                     [type="checkbox"]:not(:checked),
```

Comment:
not(:checked) is the opposite of :checked right ?
If so why not just writing [type="checkbox"] ?